fastlane/Fastfile

```diff
@@ -118,12 +119,11 @@ end
     ios_bump_version_release()
     new_version = ios_get_app_version()
     ios_update_release_notes(new_version: new_version)
+    get_prs_list(repository:GHHELPER_REPO, milestone: new_version, report_path:"#{File.expand_path('~')}/simplenotemacos_prs_list_#{old_version}_#{new_version}.txt")
     setbranchprotection(repository:GHHELPER_REPO, branch: "release/#{new_version}")
     setfrozentag(repository:GHHELPER_REPO, milestone: new_version)
 
     ios_localize_project()
-    ios_tag_build()
```

Comment:
This is so that we can better test #956.

Besides, it doesn't do anything:

![image](https://user-images.githubusercontent.com/1218433/118571226-e3e56e80-b7c0-11eb-832d-e144b573c5aa.png)

See https://github.com/Automattic/simplenote-macos/blob/7436282b5a6dce46bd5303426f6a77f7b6987c2e/.circleci/config.yml#L69-L83